src/test/kotlin/pl/newbies/event/EventTest.kt

```diff
@@ -732,6 +818,20 @@ class EventTest : IntegrationTest() {
             add(createEvent(authResponse, visibility = Event.Visibility.PRIVATE))
         }
 
+        @JvmStatic
+        fun vanityUrlTestCases() = listOf(
+            VanityUrlTestCase("Jak skutecznie japko", "jak-skutecznie-japko"),
+            VanityUrlTestCase("spaceonlastchar   ", "spaceonlastchar"),
+            VanityUrlTestCase("illegallastchar   <", "illegallastchar"),
+            VanityUrlTestCase("123 - testowa N4zwa", "123-testowa-n4zwa"),
+            VanityUrlTestCase("345-testowa N4zwa", "345testowa-n4zwa"),
+            VanityUrlTestCase("testowe  wydarzenie", "testowe-wydarzenie"),
+            VanityUrlTestCase("somerandomwithoutspaces", "somerandomwithoutspaces"),
+            VanityUrlTestCase("/to/do k,o'm`entarz", "todo-komentarz"),
+            VanityUrlTestCase("ąąłłó óććęę", "aallo-occee"),
```

Comment:
what about non polish's weird chars `Ää Ãã Āā Áá Àà` ?

Reply:
I've added more tests, all non-standard characters should be now handled or removed